Hymns/Tags/TagStore.swift

```diff
@@ -65,6 +66,19 @@ class TagStoreRealmImpl: TagStore {
     func queryTagsForHymn(hymnIdentifier: HymnIdentifier) -> Results<TagEntity> {
         return realm.objects(TagEntity.self).filter(NSPredicate(format: "primaryKey CONTAINS[c] %@", ("\(hymnIdentifier.hymnType):\(hymnIdentifier.hymnNumber):\(hymnIdentifier.queryParams ?? [String: String]())")))
     }
+
+    //A workaround to the fact that Realm does not yet support querying unique values. Force cast should never fail since the realm object will always be a string and castable into a string array.
+    //swiftlint:disable force_cast
```

Comment:
It seems like this is supported: https://stackoverflow.com/a/49523428/1907538

Reply:
Nice! Great find. I made the change